### PR TITLE
Adding how to get involved information to project README

### PR DESCRIPTION
**Pull request by missBerg**
Adding information on how to get involved in the project to project README.

### CODE REVIEW

**Review by zirain — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,2 +1,23 @@
...
+We adhere to the [CNCF Code of conduct][Code of conduct]
+
+The Envoy AI Gateway team meets every Thursday.
+Please register for the meeting, add agenda points, and get involved.
+
+* [Community Meeting details][meeting]
+
+
+[meeting]: https://docs.google.com/document/d/10e1sfsF-3G3Du5nBHGmLjXw5GVMqqCvFDqp_O65B0_w/edit?tab=t.0
+[blog]: https://blog.envoyproxy.io/introducing-envoy-gateway-ad385cc59532
```
Comment (arkodg) [inline README.md:20]:
```suggestion

```

**Comment by codefromthecrypt** [comment]
Is this the outcome of this doc? maybe if so link it for historical context? https://docs.google.com/document/d/1FQN_hGhTNeoTgV5Jj16ialzaSiAxC0ozxH1D9ngCVew/edit?tab=t.0#heading=h.uuu99yemq4eo

**Comment by missBerg** [comment]
> Is this the outcome of this doc? maybe if so link it for historical context? https://docs.google.com/document/d/1FQN_hGhTNeoTgV5Jj16ialzaSiAxC0ozxH1D9ngCVew/edit?tab=t.0#heading=h.uuu99yemq4eo

It's the doc that inspired the work and brought us together 
I think we can add a background/history section in a separate PR :) 

It is covered for history in the meeting notes: https://docs.google.com/document/d/10e1sfsF-3G3Du5nBHGmLjXw5GVMqqCvFDqp_O65B0_w/edit?tab=t.0#heading=h.6nxfjwmrm5g6

**Comment by missBerg** [comment]
@codefromthecrypt added a note on the background, think we can add in more later :)

**Pull request merged**